user-org-actor/src/main/java/org/sunbird/user/service/UserServiceImpl.java

```diff
@@ -48,4 +50,15 @@ private Response saveUserAttributes(Request request) {
         }
         return response;
     }
+
+    @Override
+    public Response readUser(Request request) throws BaseException {
+        ObjectNode idNode = JsonNodeFactory.instance.objectNode();
+        ObjectNode userNode =  JsonNodeFactory.instance.objectNode();
```

Comment:
Are we going to return user data in same format as we are saving in ES(with complete sub entity data like address, education, job profile, its root org details, user_org details etc) or  returning only basic user data.

Reply:
We are okay for returning only the basic data now.